drivers/staging/vc04_services/vchiq-mmal/mmal-parameters.h

```diff
@@ -23,21 +23,21 @@
 #define MMAL_PARAMETERS_H
 
 /** Common parameter ID group, used with many types of component. */
-#define MMAL_PARAMETER_GROUP_COMMON            (0 << 16)
+#define MMAL_PARAMETER_GROUP_COMMON		(0 << 16)
```

Comment:
Here and a few other places include random whitespace changes. Those ought to be in a different patch.

Reply:
Done.